source/ETL/Data Pipeline/Jupyter Notebooks/ETL Data Validation.ipynb

```diff
@@ -0,0 +1,1740 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a4401042",
+   "metadata": {},
+   "source": [
+    "# Data Validation\n",
+    "\n",
+    "Compare data updated using the classic and modern pipeline. The current scope of validation is to ensure completeness and integrity. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "id": "c0b43b1a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import dataframe_image as dfi\n",
+    "import pyodbc\n",
+    "from sqlalchemy.engine import URL\n",
+    "import sqlalchemy as sa"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 38,
+   "id": "4f0c5bcc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "host = #Provide host\n",
+    "user = #Provide username\n",
+    "password = #Provide password\n",
+    "database = 'rdb_modern'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "id": "090a1861",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#SQLACLHEMY\n",
+    "connection_string = \"DRIVER={ODBC Driver 17 for SQL Server};SERVER=\"+host+\";DATABASE=\"+database+\";UID=\"+user+\";PWD=\"+password\n",
+    "connection_url = URL.create(\"mssql+pyodbc\", query={\"odbc_connect\": connection_string})\n",
+    "from sqlalchemy import create_engine\n",
+    "engine = create_engine(connection_url)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "id": "f1e7446a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with engine.begin() as conn:\n",
+    "    sql_rdb_mod = pd.read_sql_query(sa.text(\"Select * from rdb_modern.dbo.HEPATITIS_DATAMART hd where hd.PATIENT_UID  = '101831376'\"), conn)\n",
+    "    sql_rdb = pd.read_sql_query(sa.text(\"Select * from rdb.dbo.HEPATITIS_DATAMART hd where hd.PATIENT_UID  = '101831376'\"), conn)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "id": "63fb0cd4",
+   "metadata": {},
+   "outputs": [
```

Comment:
Will this output remain the same if we execute the above scripts on a different timeframe. If not, will be ok if we remove the output results.

Reply:
It will change if the base dataset the query is pulling from is changed and it can be removed.